lib/core.js

```diff
@@ -783,19 +783,21 @@ declare class Proxy<T> {
 declare var global: any;
 
 declare var module: {
-    exports: any;
-    require(id: string): any;
+    exports: typeof exports;
+    require(id: string): typeof exports;
     id: string;
-    filename: string;
+    filename: ?string;
```

Comment:
```sh-session
$ node
> module.filename
null
```

Reply:
Changes like this have no sense for me.
If code in a file, module.filename is always exists.